aprslib/parsing/__init__.py

```diff
@@ -138,6 +139,26 @@ def parse(packet):
 def _try_toparse_body(packet_type, body, parsed):
     result = {}
 
+    unsupported_formats = {
```

Comment:
This name map should be outside of the function. The `unused`, `reseved` names are not very descriptive.

Reply:
While the `unused` and `reserved` names aren't descriptive, I was taking those from the comment describing the unsupported types.

Reply:
`unused` and `reserved` are consistent with labeling in the [APRS spec](http://www.aprs.org/doc/APRS101.PDF) for Data Type Identifiers.